inc/taskview.class.php

```diff
@@ -478,6 +516,8 @@ public function submitForm($postvars) {
          }
          $this->update($postvars);
          Html::back();
+      } else if (isset($postvars['export_jobs'])) {
+         $this->csvExport($postvars);
```

Comment:
In task.form.php, you call this function `submitForm` before the checkright, so you need add a checkright here in the case of export